rules/prefer-prototype-methods.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+const getDocumentationUrl = require('./utils/get-documentation-url');
+const methodSelector = require('./utils/method-selector');
+const getPropertyName = require('./utils/get-property-name');
+
+const messages = {
+	'known-constructor-known-method': 'Prefer use `{{constructorName}}.prototype.{{methodName}}`.',
```

Comment:
```suggestion
	'known-constructor-known-method': 'Prefer using `{{constructorName}}.prototype.{{methodName}}`.',
```

Same with the others.